zebrad/tests/acceptance.rs

```diff
@@ -38,6 +39,9 @@ use zebrad::config::ZebradConfig;
 /// `tracing_endpoint` test failures on some machines.
 const LAUNCH_DELAY: Duration = Duration::from_secs(3);
 
+///
```

Comment:
Please don't add empty comments to silence clippy errors.

Either write a short comment, or use `#[allow(missing_docs)]`